packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -176,20 +209,73 @@ SquadPage.layoutProps = mainFeedLayoutProps;
 
 export default SquadPage;
 
-export async function getStaticPaths(): Promise<GetStaticPathsResult> {
-  return { paths: [], fallback: true };
-}
-
 interface SquadPageParams extends ParsedUrlQuery {
   handle: string;
 }
 
-export function getStaticProps({
+export async function getServerSideProps({
   params,
-}: GetStaticPropsContext<SquadPageParams>): GetStaticPropsResult<SourcePageProps> {
-  return {
-    props: {
-      handle: params.handle,
-    },
+  query,
+  res,
+}: GetServerSidePropsContext<SquadPageParams>): Promise<
+  GetServerSidePropsResult<SourcePageProps>
+> {
+  const { handle } = params;
+  const { userid: userId, cid: campaign } = query;
+
+  const setCacheHeader = () => {
+    res.setHeader(
+      'Cache-Control',
+      `public, max-age=0, must-revalidate, s-maxage=${oneHour}`,
+    );
   };
+
+  try {
+    const promises = [];
+
+    promises.push(
```

Comment:
Why don't you set this one as the default already? (Seeing it's non conditional)

Reply:
Just our typescript being funny. It then infers the type to always be Promises of result of squad query. 😆

Reply:
Ah sigh haha